src/bin/commands/account-readiness/index.ts

```diff
@@ -1,13 +1,28 @@
+import chalk from "chalk";
 import type { AwsAccountReadiness, CliCommandResponse } from "../../../types/cli";
-import { ssmParamReadiness } from "./ssm";
-import { vpcReadiness } from "./vpc";
+import { report as reportSSM } from "./ssm";
+
+export interface Report {
+  name: string;
+  isPass: boolean;
+  msg?: string;
+  errors: Map<string, string>;
+}
 
 export const accountReadinessCommand = async (props: AwsAccountReadiness): CliCommandResponse => {
   // Got a new AWS account readiness command? Add it to this list and ✨
-  const commandResponses: number[] = await Promise.all([ssmParamReadiness(props), vpcReadiness(props)]);
+  const reports = await Promise.all([reportSSM(props)]);
```

Comment:
If there are no immediate plans to add more reports here, then we could simplify the code a bit by working with a single report rather than an array?